tools/rpkg/tests/testthat/test_arrow.R

```diff
@@ -0,0 +1,295 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+library(arrow, warn.conflicts = FALSE)
+library(dplyr, warn.conflicts = FALSE)
+library(duckdb)
+library("testthat")
+library("DBI")
+skip_on_cran()
+skip_on_os("windows")
+skip_if_not_installed("arrow", "5.0.0")
+# Skip if parquet is not a capability as an indicator that Arrow is fully installed.
+skip_if_not(arrow::arrow_with_parquet(), message = "The installed Arrow is not fully featured, skipping Arrow integration tests")
+skip_if_not_installed("dbplyr")
+
+example_data <- tibble::tibble(
```

Comment:
`tibble` is probably not even a suggested package in DuckDB. Certainly it comes with `dplyr` and others, but maybe that complicates testing? Is there some particular reason to use tibble instead of data.frame here?